helper.js

```diff
@@ -1,14 +1,21 @@
 const fs = require("fs-extra");
 const path = require("path");
 
-const tempDirPath = path.resolve(
-  process.env.JEST_HTML_REPORTERS_TEMP_DIR_PATH || __dirname,
-  "temp"
-);
-const dataDirPath = path.resolve(tempDirPath, "./data");
-const attachDirPath = path.resolve(tempDirPath, "./images");
-
-const distDirName = `./jest-html-reporters-attach`;
+const distDirName = './jest-html-reporters-attach';
+
+const tempDirPath = () => {
```

Comment:
@Hazyzh updated.

P.S. Do you plan to move the project to TS?
P.S. Also I see a lot of different formattings around code? Is it ok to introduce some lint, prettier stuff to repo and make it consistent formatting?

Reply:
Hi @sirdir, plz do not add the Timestamp, because this variable will be using in the whole testing period time (save data when running a specific test file,  read data when finished the testing), we need to make sure this variable is consistent at all time.

P.S. Do you plan to move the project to TS?
Yes, but I don't have enough time recently.
P.S. Also I see a lot of different formattings around code? Is it ok to introduce some lint, prettier stuff to repo and make it consistent formatting?
agree. will add later.

Reply:
made force push, to clean PR history

Reply:
thanks